server/postgreSQL/connection.js

```diff
@@ -0,0 +1,11 @@
+const Pool = require('pg').Pool;
+
+const pool = new Pool({
+  user: 'dfuent',
+  host: 'localhost',
+  database: 'movies',
+  password: '',
+  port: 5432
+});
```

Comment:
Implement a dotenv to avoid hard coding credentials

Reply:
@mmcphillips dotenv.config() being used now